### ISSUE TITLE
ChromeOS Can't load recordings in playback wrapper

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-and-orbits#347]
For https://github.com/phetsims/QA/issues/505. On Lovelace (ChromeOS) when trying to load a recording onto the playback wrapper, it took a very long time to show up. When it did, it seemed to be unable to play. Chrome eventually came up with a message that the page was unresponsive. No errors were seen in the console. I was able to quickly load up the log on a subsequent try, but was still unable to play the recording.

**Comment by samreid** [comment]
@arouinfar and I agreed this would not block an upcoming GAO dev version.  @zepumph any idea how to proceed on this?

**Comment by zepumph** [comment]
@KatieWoe, I don't have a chromeOS device, so I will likely need more information from you. Can you please go to a playback wrapper and add the `?debug` query parameter, then in the console type

```
window.localStorage.clear()
```

Then load the file you are trying to playback. If you can reproduce the problem, then save the console into a log file and give it to me please.


> but was still unable to play the recording.

I assume this means no errors that time either, is that right?

**Comment by KatieWoe** [comment]
>but was still unable to play the recording.

>I assume this means no errors that time either, is that right?

Yes it does mean that. I'll comment again when I've done that test.

**Comment by KatieWoe** [comment]
I went to reproduce the original bug and I did end up getting a console error this time as it tried to play the video.
![Screenshot 2020-06-19 at 10 45 35 AM](https://user-images.githubusercontent.com/41024075/85160666-df64ae80-b21b-11ea-838a-bd6a0e2144fc.png)

**Comment by KatieWoe** [comment]
Following the instructions in https://github.com/phetsims/gravity-and-orbits/issues/347#issuecomment-646381386, I had the slow loading and failure to play. Here's the log file:
[phet-dev.colorado.edu-1592588121588.log](https://github.com/phetsims/gravity-and-orbits/files/4806037/phet-dev.colorado.edu-1592588121588.log)

**Comment by zepumph** [comment]
Thank you for the above. I really appreciate it. A couple more things that will help out tremendously

1. Could you please provide that log that caused this error here, 
2. Could you use that same log to try to reproduce this on master. Likely at a link something like http://phettest.colorado.edu/phet-io-wrappers/playback/?sim=gravity-and-orbits&debug. Please provide that log here as well.


Thanks!

**Comment by KatieWoe** [comment]
Here is the log of the recording: 
[gravtest.log](https://github.com/phetsims/gravity-and-orbits/files/4806968/gravtest.log)
Having trouble with number 2. Will pair with @zepumph on Wednesday.

**Comment by zepumph** [comment]
@KatieWoe and I are pairing, and are reproducing the problem now together, and we found that it is very laggy on chromeOS to try to play back the above log. Once we pressed play, and it said "Starting Playback. . . ", it took 120 seconds to show the splash screen. We found that this is because, for some undetermined reason, there is a lot of work being done to play this session. Even on my local computer, it is quite intensive to start the session. It took about 30 seconds of parsing.

With this in mind. It seems like the connection reset error above (which ends up causing the playback failure) is causing this issue on ChromeOS, likely because it takes longer than that timeout, whereas on my comp, it doesn't take that long (though it is still way to long). I will try to make this session run more efficiently. 

More notes for this issue:
* Some investigation on this issue should be at a high priority, to make sure that it isn't because of the recorded logs. 
* Supporting the playback wrapper on a ChromeOS is not a high priority, and I don't think that we should devote resources to that right now. That said this issue is more about performance of the Playback wrapper in general, not just on ChromeOS.

Thanks @KatieWoe for your time today!
